Source/WebCore/html/InvokerElement.idl

```diff
@@ -25,6 +25,6 @@
 
 [EnabledBySetting=InvokerAttributesEnabled]
 interface mixin InvokerElement {
```

Comment:
Shouldn't this be renamed as well?

Reply:
Would `InvokerCommandElement` be a suitable name? or would you prefer something like `CommandButtonElement`?

Reply:
I guess this part we should first discuss in the WHATWG as we might well be able to flatten it into HTMLButtonElement.

Reply:
Given the conclusion in whatwg, I'll follow up with some of these changes.